src/Silverback.Integration.Kafka/Messaging/Messages/KafkaMessageHeaders.cs

```diff
@@ -18,10 +18,5 @@ public static class KafkaMessageHeaders
         ///     Used to temporary store the target partition index for the produced message.
         /// </summary>
         public const string KafkaPartitionIndex = "x-kafka-partition-index";
-
-        /// <summary>
-        ///     The header that will be filled with the timestamp of the message consumed from Kafka.
-        /// </summary>
-        public const string TimestampKey = "x-kafka-message-timestamp";
```

Comment:
This is a breaking change, we should leave this in, with updated value (`x-message-timestamp`) and marked as obsolete.